targets/nuttx-stm32f4/jerry_main.c

```diff
@@ -346,6 +346,8 @@ int main (int argc, FAR char *argv[])
 int jerry_main (int argc, char *argv[])
 #endif
 {
+  srand ((unsigned) (jerry_port_get_current_time ()));
```

Comment:
nuttx's `jerry_port_get_current_time` implementation is a simple `return 0;`, so calling `srand` does not help a lot here